src/main/java/cz/muni/fi/pa165/bluebat/dao/ReservationDao.java

```diff
@@ -0,0 +1,40 @@
+package cz.muni.fi.pa165.bluebat.dao;
+
+import cz.muni.fi.pa165.bluebat.entity.Reservation;
+
+/**
+ * An interface for manipulation with the Reservation database table.
+ *
+ * @author Dominik Baranek <460705@mail.muni.cz/>
+ */
+public interface ReservationDao {
```

Comment:
Ještě bych přidal funkci findAll(), která by vrátila všechny rezervace, To si myslím bude dobré, když admin bude chtít vidět všechny rezervace